wagtail/admin/wagtail_hooks.py

```diff
@@ -1091,6 +1091,7 @@ def register_icons(icons):
         "rotate.svg",
         "search.svg",
         "site.svg",
+        "sliders.svg",
```

Comment:
This will also need adding to our `wagtail_icons_table.txt` when merging. I can take care of that.